package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A project that counts towards Microverse graduation.",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack"
   },
   "repository": {
```

Comment:
Hello great job with the ameliorations. However please we have one last thing (small detail) regarding the LAST comment of former reviewer. Please add these lines to your `package.json` file :
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

``

Then we'll be good to go